<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxPic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>831</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>picture</string>
    </property>
    <widget class="QLabel" name="labelPic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>811</width>
       <height>581</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxPrams">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>231</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>parameters</string>
    </property>
    <widget class="QLabel" name="labelK1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>K1</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelK2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>K2</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelK3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>K3</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelP1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>P1</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelP2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>P2</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSKEW">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>SKEW</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>FX</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>FY</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelOX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>OX</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditK1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditK2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditK3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditP1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditP2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSKEW">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFX">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFY">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditOX">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelOY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>OY</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditOY">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelAngle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>angle</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDep">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>dep</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLarge">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>large</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAbodon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>abodon</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>340</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>430</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxActs">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>460</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>save and load</string>
    </property>
    <widget class="QPushButton" name="pushButtonLoad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>loadPic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSave">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>saveParam</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
